packages/back-end/src/services/experiments.ts

```diff
@@ -241,8 +241,20 @@ export async function getManualSnapshotData(
     const metric = metricMap.get(m);
     if (!metric) return null;
 
+    const denominator =
+      metric.denominator && !isFactMetric(metric)
+        ? metricMap.get(metric.denominator)
+        : undefined;
     metricSettings[m] = {
-      ...getMetricSettingsForStatsEngine(metric, metricMap, false),
```

Comment:
seems like it would be better to make `getMetricSettingsForStatsEngine` reflect this nuance than to copy/paste most of the logic here

Reply:
We would have to build a snapshot settings object for manual snapshots which we never have. I was just lazy here because we don't care about manual snapshots, but I'll take another look to see if it's easy to build it.

Reply:
Done, drier now.